views/templates/homecategoriez-boilerplate.tpl

```diff
@@ -13,24 +13,22 @@
     <ul class="row">
         {foreach from=$categories item=category name=homeCategory}
             {assign var='categoryLink' value=$link->getcategoryLink($category->id_category, $category->link_rewrite)}
-            {assign var='imageLink' value=$link->getCatImageLink($category->link_rewrite, $category->id_category, $pic_size_type)}
+            {assign var='imageLink' value=$link->getCatImageLink($category->link_rewrite, $category->id_image, 'category_default')|escape:'html':'UTF-8'}
             <li class="col-xs-12 col-sm-6 col-lg-2">
                 <a href="{$categoryLink}" title="{$category->name|escape:html:'UTF-8'}">
-                    {if file_exists($smarty.const._PS_CAT_IMG_DIR_|cat:basename($imageLink))}
+                        {if $category->id_image} 
                         <img
                             src="{$imageLink}"
                             width="{$pic_size.width}"
                             height="{$pic_size.height}"
                             alt="{$category->name|escape:html:'UTF-8'}"
-                            title="{$category->name|escape:html:'UTF-8'}"
-                        >
+                            title="{$category->name|escape:html:'UTF-8'}">
                     {else}
                         <img
                             src="{$urls.img_cat_url|escape:'html':'UTF-8'}{$language.iso_code|escape:'html':'UTF-8'}.jpg"
                             width="{$pic_size.width}"
                             height="{$pic_size.height}"
-                            title="{l s='No image' mod='tablecombz'}"
-                        >
+                            title="{l s='No image' mod='tablecombz'}">
```

Comment:
1. There was a more friendly formatting style.
2. There is a mistake: change `tablecombz` to `homecategoriez`